### ISSUE TITLE
Feature: Add number of children species to the duplicate reports for ranks above species

### ISSUE THREAD

**Issue by gdower** [CatalogueOfLife/checklistbank#1497]
### Description

While assessing which duplicate name to block, often blocking the name with the fewest children species attached is desirable. In the current workflow the user has to open both duplicates in new tabs to find the number of children species, then remember which checkbox to select. Rendering the number of children species directly in the duplicates tool could significantly speed up the process. It would also be useful to have a select the fewest children species button. It is important that it is number of species children because sometimes there can be a pretty extensive empty classification with no species.

Here's a mock-up of what this might look like:

![image](https://github.com/user-attachments/assets/89056859-bffd-4e90-a196-7fbfdeb802b3)

**Comment by mdoering** [comment]
Good idea! Maybe even have both all descendants and only (accepted) species counts? In any case that should require an API change instead of getting the counts from the search

**Comment by gdower** [comment]
I think @yroskov might prefer only accepted species counts mainly because of limited screen space. To some degree that could be fixable if it were possible to re-arrange the columns so that the columns needed to make a decision could be dragged to the left. Maybe if it was just 1 children column presented like 30/32, with 30 being the accepted species count and 32 being the total children it would still be pretty compact?

**Comment by mdoering** [comment]
Species counts is likely the most useful one. Probably good enough and keeps it simple.

PS: Note that children usually means direct children in a tree whereas descendants means all nodes of the subtree.
